interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/conference-room.twig

```diff
@@ -61,4 +72,114 @@
+                        <input type="text" class="form-control m-2" name="fname" placeholder="{{ "First Name"|xlt }}" />
+                        <input type="text" class="form-control m-2" name="lname" placeholder="{{ "Last Name"|xlt }}" />
+                        <input type="text" class="form-control m-2" name="DOB" placeholder="{{ "Date of Birth YYYY-MM-DD"|xlt }}" />
+                        <input type="text" class="form-control m-2" name="email" placeholder="{{ "Email Address"|xlt }}" />
+                        <p>{{ "Confirm that this individual has provided permission to receive electronic communications via email"|xlt }}</p>
+                        <input type="button" class="btn btn-secondary btn-cancel-screen-action" value="{{ "Cancel" |xla }}" />
+                        <input type="button" class="btn btn-primary btn-create-patient" value="{{ "Create Patient" |xla }}" />
+                    </div>
+                </div>
+                <div class="row d-none screen secondary-screen search-patient mt-3">
+                    <div class="col-12">
+                        <input type="text" class="form-control m-2" name="pid" placeholder="{{ "External ID"|xlt }}" />
+                        <p class="m-2">{{ "Or"|xlt }}</p>
+                        <input type="text" class="form-control m-2" name="fname" placeholder="{{ "First Name"|xlt }}" />
+                        <input type="text" class="form-control m-2" name="lname" placeholder="{{ "Last Name"|xlt }}" />
+                        <input type="text" class="form-control m-2" name="DOB" placeholder="{{ "Date of Birth YYYY-MM-DD"|xlt }}" />
+                        <input type="text" class="form-control m-2" name="email" placeholder="{{ "Email Address"|xlt }} ({{ "Exact Match" }})" />
+                        <input type="button" class="btn btn-secondary btn-cancel-screen-action" value="{{ "Cancel" |xla }}" />
+                        <input type="button" class="btn btn-primary btn-invite-search" value="{{ "Search Patients" |xla }}" />
+                        <ul class="search-patient-list list-group d-none mt-3">
+                            <li class="d-none duplicate-match-row-template list-group-item list-group-item-action flex-column align-items-start">
+                                <div class="d-flex w-100 justify-content-between">
+                                    <h1><i class="fa fa-user"></i></h1>
+                                    <h5 class="mb-1">
+                                        <span class="fname"></span> <span class="lname"></span>
+                                    </h5>
+                                    <small>(<span class="pid"></span>)</small>
+                                </div>
+                                <p><span class="dob"></span> <span class="email"></span></p>
+                                <input type="button" class='btn btn-secondary btn-select-patient' value="{{ "Invite Patient" }}" />
```

Comment:
`xla`
(also for the buttons where are translating label/value, ensure not relying on the value (since it will be translating) for underlying algorithms)

Reply:
Fixed